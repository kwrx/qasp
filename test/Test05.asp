% https://en.wikipedia.org/wiki/Boolean_satisfiability_problem
% ∀x ∀y ∀z (x ∨ y ∨ z) ∧ (¬x ∨ ¬y ∨ ¬z)

%@forall
x | nx.
%@forall
y | ny.
%@forall
z | nz.
%@constraints
{x;y;z;nx;ny;nz}.
:- nx, ny, nz.
:- x, y, z.


%% Result: INCOHERENT