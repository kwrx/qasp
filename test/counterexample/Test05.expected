QASP 1.0

{x,y,z}
{nx,ny,nz}
